<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="InequalityTest-20160406174913" tests="1" time="0.670">
	<testcase classname="InequalityTest" name="test_IsClassify_1" time="0.670">
		<error message="name 'Actuval' is not defined" type="NameError">
<![CDATA[Traceback (most recent call last):
  File "C:/Users/CHRIS/PycharmProjects/Requests/Requests.py", line 25, in test_IsClassify_1
    self.assertEqual(Actuval, expected)
NameError: name 'Actuval' is not defined
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[200
[{'id': 500, 'pct': 90, 'type': 1}]
]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
